testkit/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
What do you think about the new core changelog format? As for me, it would be better to use it everywhere, so instead of "changed" section" there probably should be "bug fixes". Maybe such format is too much for our other projects, so we can list the improvements without sections?..